app/assets/stylesheets/layout/common/_footer.scss

```diff
@@ -38,19 +62,164 @@
   margin-bottom: $baseline-unit*4;
 }
 
+.l-footer-primary__social-sharing {
+  @mixin counter-inline() {
+    text-align: left;
+
+    .button {
+      position: absolute;
+    }
+
+    .social-sharing__counter {
+      position: relative;
+      top: 27.5px;
+      transform: translateY(-50%);
+      margin: 0 0 0 55px;
+      padding: 0 0 0 10px;
+    }
+
+    .footer-primary__list-item {
+      position: absolute;
+      top: 50%;
+      margin-top: -27.5px;
+    }
+  }
+
+  .button,
+  .button:active,
+  .button:focus {
+    border-bottom: none;
+    border-radius: 50%;
+    border: none;
+  }
+
+  .social-sharing__counter {
+    margin: $baseline-unit 0;
+    color: $color-black;
+  }
+
+  margin-top: 0;
+  padding-left: 0;
+  text-align: center;
+
+  .footer-primary__list-item {
+    line-height: 1;
+  }
+
+  @include respond-to($mq-s, 37.5em) {
```

Comment:
The `37.5em` should probably be referencing one of the 'max' variables in here: https://github.com/moneyadviceservice/yeast/blob/master/assets/lib/_grid.scss#L11-L19